### PR TITLE
SAAS-255 Use get-stale-packages-action

### PR DESCRIPTION
**Pull request by palash25**

### CODE REVIEW

File: .github/workflows/clean.yml [hunk 1]
```diff
@@ -45,23 +45,13 @@ jobs:
...
       - name: Collect stale package versions
-        id: collect
         env:
           ROBOT_TOKEN: ${{ secrets.ROBOT_TOKEN }}
-        run: |
-          versions=$(bin/get-stale-packages)
-          length=$(echo $versions | wc -w)
-          echo ::set-output name=stale_versions::${versions}
-          echo ::set-output name=length::${length}
+        uses: percona-platform/get-stale-packages-action@SAAS-255-Package-Get-Stale-Packages
```
Comment (palash25) [inline .github/workflows/clean.yml:51]: change version tag after release

**Comment by palash25** [comment]
`get-stale-packages: Environment variable ROBOT_TOKEN is empty.`

@AlekSi is the ROBOT_TOKEN not set for this repo?

**Comment by AlekSi** [comment]
No, because that repository is public… I will think about it

**Comment by AlekSi** [comment]
It is not possible to delete public packages

**Pull request closed without merging**